app/controllers/admin/tickets_controller.rb

```diff
@@ -38,21 +38,50 @@ def update
       end
     end
 
+    def give
+      ticket_purchase = @ticket.ticket_purchases.new(gift_ticket_params)
```

Comment:
After using this for a conference, the workflow works pretty well! But, this leads to a somewhat weird scenario when a user has an _unpaid_ ticket. Sometimes we want to give a sponsored ticket to someone who started to go through the purchase process and when this happens it requires console access to first delete their ticket. I wonder if it makes sense to have a "replace" option, or if an unpaid ticket exists to just update it to paid with a value of $0.